ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-api-gnmi.j2

```diff
@@ -13,7 +13,31 @@ management api gnmi
       vrf {{ vrf }}
 {%         endif %}
 {%     endfor %}
-{%     if management_api_gnmi.octa is arista.avd.defined %}
+{%     if management_api_gnmi.octa is defined %}
```

Comment:
we should use `arista.avd.defined` because it also tests for `None` [defined-test](https://avd.sh/en/latest/plugins/index.html#defined-test)

Reply:
fixed